flink-kubernetes/src/test/java/org/apache/flink/kubernetes/kubeclient/resources/KubernetesLeaderElectorITCase.java

```diff
@@ -110,4 +112,44 @@ void testMultipleKubernetesLeaderElectors() throws Exception {
             kubernetesExtension.getFlinkKubeClient().deleteConfigMap(leaderConfigMapName).get();
         }
     }
+
+    @Test
+    void testClusterConfigMapLabelsAreSet() throws Exception {
+        final Configuration configuration = kubernetesExtension.getConfiguration();
+
+        final String leaderConfigMapName =
+                LEADER_CONFIGMAP_NAME_PREFIX + System.currentTimeMillis();
```

Comment:
fyi: This part of the `KubernetesLeaderElectorITCase` was touched by https://github.com/apache/flink/pull/24132/commits/6bf01b927c63bc6087c02b15aed156c6f3f74a8a in the [FLINK-34007 PR](https://github.com/apache/flink/pull/24132/files). It will move the ConfigMap lifecycle management out of the test methods. ...just in case FLINK-34007 is going to get merged first.